docs/API.md

```diff
@@ -99,7 +99,7 @@ The result of `yield take(pattern)` is an action object being dispatched.
 
 - If it is a String, the action is matched if `action.type === pattern` (e.g. `take(INCREMENT_ASYNC)`)
 
-- If it is an array, each item in the array is matched with aforementioned rules, so the mixed array of strings and function predicates is supported. The most common use case is an array of strings though, so that `action.type` is matched against all items in the array (e.g. `take([INCREMENT, DECREMENT])` and that would match either actions of type `INCREMENT` or `DECREMENT`).
+- If it is an array, each item in the array is matched with aforementioned rules, so the mixed array of strings and function predicates is supported. The most common use case is an array of strings though, so that `action.type` is matched against all items in the array (e.g. `take([INCREMENT, DECREMENT])`) and that would match either actions of type `INCREMENT` or `DECREMENT`).
```

Comment:
It looks like the actual `)` is at the end of this line, right?

Reply:
Yes!, there is, I uncommited this
there is another parenthese unclosed yet